lib/ocpp/v201/charge_point.cpp

```diff
@@ -1372,6 +1392,186 @@ void ChargePoint::handle_unlock_connector(Call<UnlockConnectorRequest> call) {
     this->send<UnlockConnectorResponse>(call_result);
 }
 
+void ChargePoint::handle_trigger_message(Call<TriggerMessageRequest> call) {
+    const TriggerMessageRequest& msg = call.msg;
+    TriggerMessageResponse response;
+    Evse* pEvse = nullptr;
```

Comment:
style wise we prefer snake_case here

Reply:
I tried to use snake case but this was just a force of habit I guess. Will change, thanks :+1: